triplea_maps.yaml

```diff
@@ -88,10 +88,10 @@
   version: 1
   img: http://tripleamaps.sourceforge.net/images/TripleA_total_world_war_mini.png
   description: |
-    <br><em><b><font size="6">Created by Rolf Larsson and Hepster</font></b></em>
+    <br><em><b style="font-size: 6px;">Created by Rolf Larsson and Hepster</b></em>
```

Comment:
I'm not sure HTML `<font size="6">` is equivalent to CSS `font-size: 6px`.  According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font), the `size` attribute seems to be a browser-specific predefined size, not necessarily pixels.  Take a look at [this](http://jsbin.com/cenumiyume/1/edit?html,output) JSBin that demonstrates the difference.